[BUG] Panic while reading configs

**Describe the bug**
Agent fails due to missing config files, and https://github.com/goraz/onion bug :)

```
[root@erclin6db18 log]# cat ercole-agent.log
panic: runtime error: index out of range [-1]

goroutine 19 [running]:
github.com/goraz/onion/utils.MergeLayersData({0xf83b40, 0x988f40, 0xc000042c01})
        /go/pkg/mod/github.com/goraz/onion@v0.1.3/utils/utils.go:7 +0x74
github.com/goraz/onion/onionwriter.MergeLayersOnion(0xc00002a020)
        /go/pkg/mod/github.com/goraz/onion@v0.1.3/onionwriter/onion_writer.go:28 +0x1e
github.com/goraz/onion/onionwriter.DecodeOnion(0xc13b60, {0x963800, 0xc000078160})
        /go/pkg/mod/github.com/goraz/onion@v0.1.3/onionwriter/onion_writer.go:34 +0x6f
github.com/ercole-io/ercole-agent/v2/config.ReadConfig({_, _})
        /root/project/config/config.go:139 +0x2ff
main.(*program).run(0xc00021ed20)
        /root/project/main.go:56 +0xe5
created by main.(*program).Start
        /root/project/service.go:52 +0x5b
[root@erclin6db18 log]#
```

**Expected behavior**
Since the config file `/opt/ercole-agent/config.json` is present, is must use it!

--- a/config/config.go
+++ b/config/config.go
@@ -125,10 +125,10 @@ func ReadConfig(log logger.Logger) Configuration {
 
 		layers = addFileLayers(log, layers, configFile)
 	} else {
-		layers = addFileLayers(log, layers, "opt/ercole-agent/config.json")
-		layers = addFileLayers(log, layers, "usr/share/ercole-agent/config.json")
-		layers = addFileLayers(log, layers, "etc/ercole-agent/ercole-agent.json")
-		layers = addFileLayers(log, layers, "etc/ercole-agent/conf.d/*.json")
+		layers = addFileLayers(log, layers, "/opt/ercole-agent/config.json")
+		layers = addFileLayers(log, layers, "/usr/share/ercole-agent/config.json")
+		layers = addFileLayers(log, layers, "/etc/ercole-agent/ercole-agent.json")
+		layers = addFileLayers(log, layers, "/etc/ercole-agent/conf.d/*.json")
 		layers = addFileLayers(log, layers, "./config.json")
 	}
 

--- a/main.go
+++ b/main.go
@@ -48,7 +48,7 @@ type program struct {
 }
 
 func (p *program) run() {
-	confLog, err := logger.NewLogger("CONFIG")
+	confLog, err := logger.NewLogger("CONFIG", logger.LogLevel(logger.DebugLevel))
 	if err != nil {
 		log.Fatal("Can't initialize CONFIG logger: ", err)
 	}

